File: python_work/0708/Circle.py
from point import Point

class Circle:
    def __init__(self,x,y,r):
        print("일로 온다."x,y,r)
        self.p = Point(x,y) # self.사용시 자동적으로 p=Point()생성.
        self.r = r

    def showInfo(self):
        self.p.showInfo()
        print(f"반지름 = {self.r}")


c = Circle(2,2,4) #생성자 호출(__nint__)
c.showInfo()

File: java_work/ex0707/src/ex0707/Circle.java
package ex0707;

class Point {
	int xPos, yPos;

	//생성자 자동 만들기 alt shift s -> o
	public Point(int xPos, int yPos) {
		this.xPos = xPos;
		this.yPos = yPos;
	}

	public void showPointinfo() {
		System.out.println("[" + xPos + "," + yPos + "]");
	}
}
public class Circle { //클래스 명이랑 같아야 됨.
	Point point;
	int r;
	public Circle(int i, int j, int k) {
		point = new Point(i,j);
		this.r = k;
	}
	private void showCircleInfo() {
		System.out.println("반지름 r = "+this.r);
		point.showPointinfo();
	}
	public static void main(String[] args) { //제일 처음 시작
		Circle circle = new Circle(2, 2, 4);
		circle.showCircleInfo(); //creat method를 이용해 만들기(빨간줄 위에 마우스 올리면 뜸)
	}
}
/*
 * 
 */